DSESwift/DataPoint.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  DataPoint.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 16/09/15.
+//
+//
+
+import Foundation
+
+class DataPoint{
+    var sensorId = -1
+    private(set) var value = ""
+    var date = NSDate()
+    var synced = false
+
+    init(sensorId: Int, value: String, date: NSDate, synced: Bool) {
+        self.sensorId = sensorId
+        self.value = value
```

Comment:
@josdejong  @excaliburnan public getDataPoints is implemented in Sensor class.(https://github.com/senseobservationsystems/sense-ios-library/pull/98#discussion-diff-40435178R78). So the user can always get dataType from Sensor class, as long as they have access to getDataPoints method. I will leave this for now.